schema.cc

```diff
@@ -1760,7 +1760,7 @@ column_computation_ptr column_computation::deserialize(bytes_view raw) {
 
 bytes legacy_token_column_computation::serialize() const {
     rjson::value serialized = rjson::empty_object();
-    rjson::add(serialized, "type", rjson::from_string("token"));
+    rjson::add(serialized, "type", rjson::from_string(sstring("token")));
```

Comment:
I don't think it copies it, sstring constructor does `reinterpret_cast`, and then from_string passes it by const reference. Although perhaps I missed one thing, that it uses strlen while string_view does traits_type::length so that may be more efficient if strlen can't be optimized.

Goal of this particular line change was just to select the right overload explicitly.

Due to abovementioned strlen I will change sstring to string_view.

Reply:
The document introducing std::string_view, https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n3921.html, mentions that "both gcc and clang optimize strlen("Literal") into a constant, making the simple, safe code as efficient as the template."

Reply:
true, it does copy, I didn't follow the code to the deepest call